### ISSUE TITLE
Minimap does not show after I installed Nuclide

### ISSUE THREAD

**Issue by yookoala** [facebook/nuclide#986]
### Issue and Steps to Reproduce
1. Install [minimap](https://atom.io/packages/minimap).
2. Install nuclide.
3. Toggle minimap to open with Ctrl-Shift-P command.

### Expected Behavior
minimap should come up on the right side by default

### Actual Behavior
Nothing visible happend

### Versions
* Atom: 1.12.7
* Nuclide: 0.197.0
* Client OS: Ubuntu
* Server OS (optional):

### Additional Details
* Installed packages (`apm ls --installed`):
```
Community Packages (16) /home/user/.atom/packages
├── atom-touch-events@0.23.0
├── editorconfig@2.0.5
├── git-control@0.8.2
├── go-debug@1.0.3
├── go-plus@5.1.0
├── go-signature-statusbar@1.2.0
├── jsonlint@1.1.4
├── language-docker@1.1.7
├── linter@1.11.18
├── linter-docker@0.1.4
├── linter-js-yaml@1.2.6
├── linter-jsonlint@1.3.0
├── merge-conflicts@1.4.4
├── minimap@4.26.4
├── nuclide@0.197.0
└── platformio-ide-terminal@2.2.2

└── (empty)
```

**Comment by jgebhardt** [comment]
Resolved here: https://github.com/atom-minimap/minimap/issues/561#issuecomment-273516974

TL;DR minimap does not support Atom <1.13.0, and OP was on 1.12.7.
